js/views/pageNavVw.js

```diff
@@ -91,17 +96,48 @@ module.exports = Backbone.View.extend({
   },
 
   handleSocketMessage: function(response) {
-    var data = JSON.parse(response.data);
-    if(data.id == this.socketNotificationID){
-      console.log(data);
-    }
-  },
+    var data = JSON.parse(response.data),
+        username,
+        avatar,
+        n;
+
+    if (data.hasOwnProperty('notification')) {
+      n = data.notification;
+      username = n.handle ? n.handle : n.guid.substring(0,10) + '...';
+      avatar = n.image_hash ? app.serverConfig.getServerBaseUrl + '/get_image?hash=' +
+        n.image_hash + '&guid=' + n.guid : 'imgs/defaultUser.png';
+
```

Comment:
In this case, I'm just talking about adding it to the temporary model you're creating for the individual notification. It doesn't get reused anywhere. Even if it did, the original handle and guid would still be there. But it's also fine to leave the truncation in the template.